frontend/src/pages/modelServing/screens/projects/NIMServiceModal/ManageNIMServingModal.tsx

```diff
@@ -153,6 +160,9 @@ const ManageNIMServingModal: React.FC<ManageNIMServingModalProps> = ({
   const [storageClassName, setStorageClassName] = React.useState(
     deployedStorageClassName || defaultStorageClassName,
   );
+  const [pvcMode, setPvcMode] = React.useState<PVCMode>('create-new');
+  const [existingPvcName, setExistingPvcName] = React.useState<string>('');
+  const [modelPath, setModelPath] = React.useState<string>('/mnt/models/cache');
```

Comment:
Minor refactor, since `modelPath` string ('/mnt/models/cache') appears in multiple places maybe consider creating a constant at the top of the file and use DEFAULT_MODEL_PATH everywhere relevant?
```suggestion
  const DEFAULT_MODEL_PATH = '/mnt/models/cache';
  const [modelPath, setModelPath] = React.useState(DEFAULT_MODEL_PATH);

```

Reply:
Great suggestion!
Thanks!